README.md

```diff
@@ -62,8 +62,9 @@ $ scss-bundle -c scss-bundle.config.json
 
 ```json
 {
-    "entry": "./src/main.scss",
-    "dest": "bundled.scss"
+  "entry": "./src/main.scss",
+  "dest": "./bundled.scss",
+  "watch": "./src/*.scss"

```

Comment:
Added watch here to help clarify it's intended use as a PATH to watch, not a watch flag.